diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index 0039112..8e053b2 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -297,10 +297,10 @@ install(FILES
     ${CMAKE_BINARY_DIR}/exiv2lib_export.h
     DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/exiv2)
 
-install(EXPORT exiv2Targets
-    DESTINATION "${CMAKE_INSTALL_LIBDIR}/cmake/exiv2"
-    NAMESPACE Exiv2::
-)
+# install(EXPORT exiv2Targets
+#     DESTINATION "${CMAKE_INSTALL_LIBDIR}/cmake/exiv2"
+#     NAMESPACE Exiv2::
+# )
 
 install(
     FILES
